tests/testthat/test-git.R

```diff
@@ -146,6 +146,50 @@ test_that("can get commit history for a branch", {
   expect_true(commits$id != other_commits$id)
 })
 
+test_that("git commits lists merge commit", {
+  testthat::skip_on_cran()
+  ## If we have a case like
+  ## master  (A)-(B)       #nolint
```

Comment:
this is fine, I've found myself doing things like:

```
## > master (A)-(B)
```

to skip this lint check on false positives. It's annoying but it does also catch lots of good true positives left in from debugging etc.